idea-plugin/build.gradle.kts

```diff
@@ -58,4 +62,9 @@ tasks {
         ideVersions(properties("plugin.verifier.ide.versions"))
         downloadDirectory("${project.buildDir}/pluginVerifier/ides")
     }
+
+    runIde {
```

Comment:
Why this change?

Reply:
It was convenient to always publish the latest Gradle plugin for dev iterations, but now thinking about it, I'm not sure if it's such a universally good idea, so I will revert the change